streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -523,13 +527,13 @@ public void shouldInvokeUserDefinedGlobalStateRestoreListener() throws Exception
 
         final Map<String, Object> kafkaStreams1Configuration = mkMap(
             mkEntry(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory(appId).getPath() + "-ks1"),
-            mkEntry(StreamsConfig.consumerPrefix(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG), appId + "-ks1"),
```

Comment:
ah, yeah, good catch -- we should definitely not be using the `group.instance.id` config in tests, it turns on static membership which will delay the test at best (and mess with it at worst)